tests/exchange/test_exchange.py

```diff
@@ -2186,7 +2190,7 @@ def test_cancel_order_dry_run(default_conf, mocker, exchange_name):
     assert exchange.cancel_order(order_id='123', pair='TKN/BTC') == {}
     assert exchange.cancel_stoploss_order(order_id='123', pair='TKN/BTC') == {}
 
-    order = exchange.buy('ETH/BTC', 'limit', 5, 0.55, 'gtc')
+    order = exchange.create_order('ETH/BTC', 'limit', "sell", 5, 0.55, 'gtc')
```

Comment:
This should be buy, not sell (according to the old version) - although for this test, the side is not really relevant.

```suggestion
    order = exchange.create_order('ETH/BTC', 'limit', "buy", 5, 0.55, 'gtc')
```